content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -109,13 +110,101 @@ Each HTTP rule contains the following information:
 A default backend is often configured in an Ingress controller to service any requests that do not
 match a path in the spec.
 
-### Default Backend
+### Path Types
+
+Each path in an Ingress has a corresponding path type. There are three supported
+path types:
+
+* **ImplementationSpecific:** With this default path type, matching is up to the
+  IngressClass. Implementations can treat this as a separate PathType or treat
+  it identically to Prefix or Exact path types
+
+* **Exact:** Matches the URL path exactly and with case sensitivity.
+
+* **Prefix:** Matches based on a URL path prefix split by `/`. Matching is case
+  sensitive and done on a path element by element basis. A path element refers
+  to the list of labels in the path split by the `/` separator. A request is a
+  match for path _p_ if every _p_ is an element-wise prefix of _p_ of the
+  request path.
+    {{< note >}}
+    If the last element of the path is a substring of the
+    last element in request path, it is not a match (for example:
+    `/foo/bar` matches`/foo/bar/baz`, but does not match `/foo/barbaz`).
+    {{< /note >}}
+
+#### Multiple Matches
+In some cases, multiple paths within an Ingress will match a request. In those
+cases precedence will be given first to the longest matching path. If two paths
+are still equally matched, precedence will be given to paths with an exact path
+type over prefix path type.
+
+### Backends
+A backend specifies the endpoints that a request should be routed to. A Service
+can be referenced using the `serviceName` and `servicePort` fields of a backend.
+Alternatively, other types of resources can be referenced with the `resource`
+object reference field.
+
+Backends defined as part of a path definition describe how traffic matching that
+path should be routed. Backends defined as part of a path definition describe
```

Comment:
Nit: I found myself rereading the second sentence a few times to properly understand what it meant. Offhand I don't have a suggestion yet, but I think we might be able to rearrange/drop a few words to increase clarity.

Reply:
Yeah I was trying to communicate the difference in meaning between `spec.backend` and `spec.rules.paths.backend` and could not find a good way to write it. Thanks for pointing out that the backend changes will really only be usable in 1.19 - because of that I've just reverted this back to the original text. We can solve this in the next release.